webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala

```diff
@@ -204,6 +219,8 @@ final case class ProjectsResponderADMLive(
       projectRestrictedViewSettingsGetADM(identifier)
     case ProjectRestrictedViewSettingsGetRequestADM(identifier) =>
       projectRestrictedViewSettingsGetRequestADM(identifier)
+    case ProjectRestrictedViewSettingsSetRequestADM(iri, user, size) =>
+      setProjectRestrictedViewSettings(iri, user, size)
```

Comment:
The `ProjectRestrictedViewSettingsSetRequestADM` is not used anywhere else and the pattern here will never match.

IMHO, we should delete `ProjectRestrictedViewSettingsSetRequestADM` and this match.